### ISSUE TITLE
Implement timestamping

### ISSUE THREAD

**Issue by CaptainFlint** [rhinstaller/pesign#14]
Microsoft signtool allows timestamped signing (/t URL and /tr URL), so that signed file remains valid even after original certificate expires. In Linux world one can sign+timestamp PE files using osslsigncode tool, but it does not support working with PKCS11 databases (and therefore with hardware tokens). It would be nice if pesign supported timestamping as well.

**Comment by vathpela** [comment]
Yeah, definitely on the TODO list.  Thanks for making sure it's filed here.

**Comment by euspectre** [comment]
Any updates on this? Looks like pesign still does not allow timestamped signing.
